buildbot/configure.py

```diff
@@ -21,17 +21,25 @@ def do_configure(args):
     ocl_header_dir = os.path.join(abs_obj_dir, "OpenCL-Headers")
     icd_loader_lib = os.path.join(abs_obj_dir, "OpenCL-ICD-Loader", "build")
     llvm_targets_to_build = 'X86'
-    llvm_enable_projects = 'clang;llvm-spirv;sycl;opencl-aot;xpti;libdevice'
+    llvm_external_projects = 'sycl;llvm-spirv;opencl-aot;libdevice'
+    llvm_enable_projects = 'clang;llvm-spirv;sycl;opencl-aot;libdevice'
     libclc_targets_to_build = ''
     sycl_build_pi_cuda = 'OFF'
     sycl_werror = 'ON'
     llvm_enable_assertions = 'ON'
     llvm_enable_doxygen = 'OFF'
     llvm_enable_sphinx = 'OFF'
     llvm_build_shared_libs = 'OFF'
+    sycl_enable_xpti_tracing = 'ON'
 
     icd_loader_lib = os.path.join(icd_loader_lib, "libOpenCL.so" if platform.system() == 'Linux' else "OpenCL.lib")
 
+    if not args.use_libcxx:
+        llvm_enable_projects += ';xpti'
+        llvm_external_projects += ';xpti'
```

Comment:
I suggest moving this code to lines 24-25 - closer to variables initialization.

I also suggest moving sycl_enable_xpti_tracing initialization before llvm_enable_projects and llvm_external_projects  to re-use it for variables initialization.

Reply:
Done.